### PR TITLE
PODS-2611 : Scheme variances - establishers - use 'newlyadded' flag for establishers and render check your answers

### PR DESCRIPTION
**Pull request by abhijeetardale**
PODS-2611 : Scheme variances - establishers - use 'newlyadded' flag for establishers and render check your answers

### CODE REVIEW

File: app/identifiers/register/establishers/company/IsCompanyDormantId.scala [hunk 1]
```diff
@@ -16,15 +16,48 @@
...
+import viewmodels.AnswerRow
 
 case class IsCompanyDormantId(index: Int) extends TypedIdentifier[DeclarationDormant] {
   override def path: JsPath = EstablishersId(index).path \ IsCompanyDormantId.toString
 }
 
-object IsCompanyDormantId {
+object IsCompanyDormantId extends Enumerable.Implicits {
+
   override def toString: String = "isCompanyDormant"
```
Comment (madhuranakate) [inline app/identifiers/register/establishers/company/IsCompanyDormantId.scala]: Dormant pages are bypassed in Variations journey, hence this code is not required

File: app/identifiers/register/establishers/partnership/IsPartnershipDormantId.scala [hunk 2]
```diff
@@ -30,9 +32,28 @@ case class IsPartnershipDormantId(index: Int) extends TypedIdentifier[Declaratio
 object IsPartnershipDormantId extends Enumerable.Implicits {
   override def toString: String = "isPartnershipDormant"
 
-  implicit val cya: CheckYourAnswers[IsPartnershipDormantId] =
-    IsDormantCYA(
-      label = "messages__partnership__checkYourAnswers__isDormant",
-      changeIsDormant = "messages__visuallyhidden__partnership__dormant"
-    )()
+  val label: String = "messages__partnership__checkYourAnswers__isDormant"
```
Comment (madhuranakate) [inline app/identifiers/register/establishers/partnership/IsPartnershipDormantId.scala]: Dormant pages are bypassed in Variations journey, hence this code is not required

Comment (abhijeetardale) [reply app/identifiers/register/establishers/partnership/IsPartnershipDormantId.scala]: reverted

File: app/identifiers/register/establishers/company/IsCompanyDormantId.scala (hunk 1 above)
Comment (abhijeetardale) [reply app/identifiers/register/establishers/company/IsCompanyDormantId.scala]: reverted

**Pull request merged**